### PR TITLE
Ac 4788: Bring models up to date with accelerate

### PR DESCRIPTION
**Pull request by mc-nathan**
Work done:
- Add Organization models and updated Startup model including migrations
- Cleaned up the text output for `make help`
- Added support for `make coverage`
- Provided full coverage for all accelerator.models
- Had to add test_runner to handle SimpleUser being unmanaged by default (per impact-api).
- Added timezone related variables to settings to get coverage working.

### CODE REVIEW

**Comment by mc-nathan** [comment]
Tests under python 2.7 fixed.  Tox runs clean for me now.

File: accelerator/tests/test_startup.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+from .factories.startup_factory import StartupFactory
+from unittest import skip
+
+
+# For some reason running StartupFactory() triggers an
+# error that reports:
+#
+#   django.db.utils.OperationalError: no such table: auth_user
+#
+# AC-4806 is a bug ticket to fix this.
```
Comment (yotammanor) [inline accelerator/tests/test_startup.py]: Can't find such a ticket.

File: Makefile [hunk 2]
```diff
@@ -16,21 +16,24 @@ targets = \
...
+  "test - Run tests. To run a subset of tests:" \
+  "\tmake test TESTS='accelerator.tests.test_currency accelerator.tests.test_startup'" \
+  "tox - Run tox to run tests on all supported configurations." \
   "uninstall - Removes the package from the local virtuanlenv." \
   "" \
-  "Note: various targets automatically create a python virtualenv if needed."
+  "Note: various targets automatically create a python virtualenv, venv." \
+  "You can us it in your shell by running: 'source venv/bin/activate'"
 
 # TODO: Figure out if we can run a single test:
```
Comment (yotammanor) [inline Makefile]: This line should be removed (since the single test was just added)

Comment (mc-nathan) [reply Makefile]: Fixed.

**Pull request merged**